lib/generators/curly/scaffold/templates/presenter_base.rb.erb

```diff
@@ -0,0 +1,23 @@
+module <%= plural_table_name.capitalize %>
+  class PresenterBase < Curly::Presenter
```

Comment:
I would remove this in favor of just duplicating those methods – it's much easier to follow generated code when there isn't any inheritance to consider, and these methods aren't that complicated.